modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_folder.jsp

```diff
@@ -380,78 +380,76 @@ renderResponse.setTitle(headerTitle);
 		}
 	}
 
-	Liferay.provide(
-		window,
-		'<portlet:namespace />selectFileEntryType',
-		function (fileEntryTypeId, fileEntryTypeName) {
-			var A = AUI();
+	window['<portlet:namespace />selectFileEntryType'] = function (
+		fileEntryTypeId,
+		fileEntryTypeName
+	) {
+		var A = AUI();
 
-			var searchContainer = Liferay.SearchContainer.get(
-				'<portlet:namespace />dlFileEntryTypesSearchContainer'
-			);
+		var searchContainer = Liferay.SearchContainer.get(
+			'<portlet:namespace />dlFileEntryTypesSearchContainer'
+		);
 
-			var fileEntryTypeLink =
-				'<a class="modify-link" data-rowId="' +
-				fileEntryTypeId +
-				'" href="javascript:;"><%= UnicodeFormatter.toString(removeFileEntryTypeIcon) %></a>';
+		var fileEntryTypeLink =
+			'<a class="modify-link" data-rowId="' +
+			fileEntryTypeId +
+			'" href="javascript:;"><%= UnicodeFormatter.toString(removeFileEntryTypeIcon) %></a>';
 
-			<c:choose>
-				<c:when test="<%= workflowEnabled %>">
-					var restrictionTypeWorkflow = A.one(
-						'#<portlet:namespace />restrictionTypeWorkflow'
-					);
+		<c:choose>
+			<c:when test="<%= workflowEnabled %>">
+				var restrictionTypeWorkflow = A.one(
+					'#<portlet:namespace />restrictionTypeWorkflow'
+				);
 
-					restrictionTypeWorkflow.hide();
+				restrictionTypeWorkflow.hide();
 
-					var workflowDefinitions =
-						'<%= UnicodeFormatter.toString(workflowDefinitionsBuffer) %>';
+				var workflowDefinitions =
+					'<%= UnicodeFormatter.toString(workflowDefinitionsBuffer) %>';
 
-					workflowDefinitions = workflowDefinitions.replace(
-						/LIFERAY_WORKFLOW_DEFINITION_FILE_ENTRY_TYPE/g,
-						'workflowDefinition' + fileEntryTypeId
-					);
+				workflowDefinitions = workflowDefinitions.replace(
+					/LIFERAY_WORKFLOW_DEFINITION_FILE_ENTRY_TYPE/g,
+					'workflowDefinition' + fileEntryTypeId
+				);
 
-					<portlet:namespace />documentTypesChanged = true;
+				<portlet:namespace />documentTypesChanged = true;
 
-					searchContainer.addRow(
-						[fileEntryTypeName, workflowDefinitions, fileEntryTypeLink],
-						fileEntryTypeId
-					);
-				</c:when>
-				<c:otherwise>
-					searchContainer.addRow(
-						[fileEntryTypeName, fileEntryTypeLink],
-						fileEntryTypeId
-					);
-				</c:otherwise>
-			</c:choose>
+				searchContainer.addRow(
+					[fileEntryTypeName, workflowDefinitions, fileEntryTypeLink],
+					fileEntryTypeId
+				);
+			</c:when>
+			<c:otherwise>
+				searchContainer.addRow(
+					[fileEntryTypeName, fileEntryTypeLink],
+					fileEntryTypeId
+				);
+			</c:otherwise>
+		</c:choose>
 
-			searchContainer.updateDataStore();
+		searchContainer.updateDataStore();
 
-			var select = A.one('#<portlet:namespace />defaultFileEntryTypeId');
+		var select = A.one('#<portlet:namespace />defaultFileEntryTypeId');
```

Comment:
Also seem to depend on `aui-node` here without declaring it (since you removed the `liferay-search-container` dependency that would bring it transitively.